src/components/User/Limits/components/Limits.Manager.vue

```diff
@@ -319,13 +310,39 @@
         }
         this.isPending = false
       },
+      async removeLimits (limits) {
+        this.isPending = true
 
+        if (limits.id === 0) {
+          this.isPending = false
+          return false
+        }
+        if (!await confirmAction({ title: 'Delete the limits?' })) return
```

Comment:
1. Add limit type and scope, like "Delete global payment limits? This action cannot be undone.", "Delete ACCOUNT_EMAIL payment limits? This action cannot be undone", "Delete syndicate’s payment limits? This action cannot be undone".
2. Since the logic is going to complicate a lot, extract to a function